aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditHorizontalExample.js

```diff
@@ -46,28 +46,23 @@ export const NameValueListEditHorizontalExample = () => {
 
   return (
     <Box gap="medium">
-      <Box
-        justify="between"
-        width="large"
-        direction="row-responsive"
-        align="start"
-        gap="medium"
-      >
-        <Box gap="xsmall">
-          <Heading level={2} size="small" margin="none">
-            Company Information
-          </Heading>
-          <Text size="large">Add your company information.</Text>
-        </Box>
-        {!edit && (
-          <Button
-            secondary
-            label="Edit"
-            onClick={() => setEdit(true)}
-            fill={['xsmall', 'small'].includes(size) ? 'horizontal' : undefined}
-          />
-        )}
-      </Box>
+      <PageHeader
+        title="Company Information"
+        subtitle="Add your company information."
+        actions={
+          !edit && (
+            <Button
+              secondary
+              label="Edit"
+              onClick={() => setEdit(true)}
+              fill={
+                ['xsmall', 'small'].includes(size) ? 'horizontal' : undefined
+              }
```

Comment:
We should look at this behavior and make the example mobile friendly. Overflow menu at certain breakpoint?

New
![Screen Shot 2022-05-04 at 5 07 37 PM](https://user-images.githubusercontent.com/1756948/166839738-8376dd5f-ffa1-416e-a3c7-d4e81da1f362.png)

Old
![Screen Shot 2022-05-04 at 5 07 49 PM](https://user-images.githubusercontent.com/1756948/166839731-b20545f7-eafc-4f0e-8d4e-375eb78b9a4a.png)

Reply:
Good call, having a look.

Reply:
☝️ sorry didn't realize that you already commented on this

Reply:
Done.